lib/index.d.ts

```diff
@@ -0,0 +1,600 @@
+import { OrderItem } from "./parsers/orderItems";
+
+type InitParams = {
+    accessKeyId: string;
+    authToken?: string;
+    host?: string;
+    merchantId: string;
+    port?: number;
+    region?: string;
+    secretAccessKey: string;
+};
+
+// cSpell: disable
+type MWS_MARKETPLACES = {
+    AU: 'A39IBJ37TRP1C6',
+    BR: 'A2Q3Y263D00KWC',
+    CA: 'A2EUQ1WTGCTBG2',
+    CN: 'AAHKV2X7AFYLW',
+    DE: 'A1PA6795UKMFR9',
+    ES: 'A1RKKUPIHCS9HS',
+    FR: 'A13V1IB3VIYZZH',
+    IN: 'A21TJRUUN4KGV',
+    IT: 'APJ6JRA9NG5V4',
+    JP: 'A1VC38T7YXB528',
+    MX: 'A1AM78C64UM0Y8',
+    TR: 'A33AVAJ2PDY3EV',
+    UK: 'A1F83G8C2ARO7P',
+    US: 'ATVPDKIKX0DER',
+};
+// cSpell: enable
+
+type ValueOf<T> = T[keyof T];
+type MarketplaceId = ValueOf<MWS_MARKETPLACES>;
+type CountryCode = keyof MWS_MARKETPLACES;
+
+type MARKET_CURRENCY = {
+    CA: 'CAD',
+    US: 'USD',
+};
+
+type CurrencyCode = ValueOf<MARKET_CURRENCY>;
+
+type MWS_ENDPOINTS = {
```

Comment:
Amazon updated the endpoints.

Reply:
yep, lots of new marketplaces and servers it seems.  adding link here so i can find it again quickly https://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html

Reply:
#179